tests/utils/dockercli/volumelifecycle.go

```diff
@@ -26,17 +26,17 @@ import (
 )
 
 // CreateVolume is going to create vsphere docker volume with given name.
-func CreateVolume(ip, name string) ([]byte, error) {
+func CreateVolume(ip, name, opts string) ([]byte, error) {
```

Comment:
Quite some test cases simply need to create a volume with default configuration. So I'd suggest to change this to a variadic function just for convenience:

   func CreateVolume(ip, name string, opts ...string) ([]byte, error)

This way we don't need to modify existing test cases.